### ISSUE TITLE
数据标签过多时显存会爆炸、预测后不出指标结果；When there are too many data labels, the video memory will explode, and the indicator results will not be displayed after prediction;

### ISSUE THREAD

**Issue by 1994su** [open-mmlab/mmrotate#333]
1. ORCNN和redet在线验证的候测试集验证完毕之后训练不再进行，而且也不出定量的指标，请问可能是什么问题呢？（这个问题在训练飞机数据集时没有问题、舰船就有问题，两个数据集数量、尺寸基本一致）
2. 我们工作站用的显卡是3080，我理解的时候显存占用跟batchsize关系比较大，但是实际和整个训练集的数据量也有关系，比如我用2000个图片和5000个图片，显存基本呈倍数关系，是否和dataloader有关系呢？

1. After the test set of ORCNN and redet online verification is completed, the training will no longer be carried out, and there will be no quantitative indicators. What may be the problem? (This problem is not a problem when training the aircraft data set, but the ship has a problem. The number and size of the two data sets are basically the same)
2. The graphics card used in our workstation is 3080. I understand that the memory usage is relatively related to the batch size, but it is actually related to the data volume of the entire training set. For example, I use 2000 pictures and 5000 pictures, and the video memory is basically in a multiple relationship. , Is it related to dataloader?

**Comment by zytx121** [comment]
As far as I know, GPU memory has nothing to do with the number of pictures but has something to do with the target number of pictures. The log shows the maximum memory in the training process, not the current memory.

BTW, you can reduce memory by modifying `gpu_assign_thr`.
https://github.com/open-mmlab/mmrotate/blob/0edd2578743559b35bab23b45058211b85ecb82c/configs/redet/redet_re50_refpn_1x_dota_ms_rr_le90.py#L40

**Comment by Telly-S** [comment]
你可以尝试写一个脚本，把数据中框特别多的图像和数据删掉，我用的12G显存，把大于1000个框的删掉了，可以跑起来了
You can try to write a script to delete the images and data with a large number of frames in the data. The 12g gpu memory I used has deleted those with more than 1000 frames and can run
